changelog/unreleased/issue-1756

```diff
@@ -0,0 +1,11 @@
+Bugfix: Mark repository files as readonly when using the local backend
+
+Files stored in a local repository are marked as readonly on non-Windows
+systems to prevent accidential file modifications. In addition, the handling
+of errors is improved which enables the local backend to work with certain
+filesystems and network mounts which do not permit modifications of file
+permissions.
```

Comment:
This paragraph is unclear. The present tense makes it unclear if you are talking about how it is after this bugfix is applied, or if you are talking about the state of things before the bugfix is applied.

I suggest writing it like this instead:

~~~
Description of the problem in past tense.

Decription of the solution implemented and how it works in present tense.
~~~

Wording like "is now" can be used to convey "after the bugfix".

Reply:
The before and after the bugfix should be much clearer now. I've added a remark that the modifications the readonly flag should help against are something external to restic. However, I'm not really happy with the wording `modifications outside of restic.` with which I ended up with.